shared/src/main/scala/mouse/feither.scala

```diff
@@ -63,7 +63,10 @@ final class FEitherOps[F[_], L, R](private val felr: F[Either[L, R]]) extends An
   def getOrElseIn[A >: R](right: => A)(implicit F: Functor[F]): F[A] =
     F.map(felr)(_.fold(_ => right, identity))
 
-  def getOrRaise[E](e: => E)(implicit F: MonadError[F, _ >: E]): F[R] =
+  def getOrRethrow(implicit F: MonadThrow[F], env: L <:< Throwable): F[R] =
+    foldF(e => F.raiseError[R](e))(F.pure)
```

Comment:
Personally, I strived not to introduce methods if there is a proper functionality in `cats` for that. Doesn't [this syntax](https://github.com/typelevel/cats/blob/main/core/src/main/scala/cats/syntax/monadError.scala#L62) work for you?

Reply:
I've completely missed this method. Closing this PR, thanks!